UnitTest/TrackingApiTest.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Application.Dto;
+using Application.Interfaces;
+using Application.Services;
+using Infrustructure.Integration;
+using Microsoft.AspNetCore.Mvc;
+using RestSharp;
+using ShipTrackingAPI.Controllers;
+using Xunit;
+
+namespace UnitTest
+{
+    public class TrackingApiTest
```

Comment:
Avoid writing unit tests for controllers.  Prefer writing thin controllers,  and  write functional tests for controllers. Unit tests for the rest.

Reply:
Also naming  we follow for tests is   as below;
Ship.Api.Unit.Tests
Ship.Service.Unit.Tests
Ship.Api.Functional.Tests
...etc